UX/ui-service/designer/Dialogs/design.md

```diff
@@ -0,0 +1,47 @@
+# Dialogs
+
+- "Dialogs" is a secondary item under the "Designer" category
+- This area is for the Service Designer and is not visible to other users
+- This area is provided so the Designer can create and edit service dialogs
+
+## Dialog Editor
+
+### Blank Dialog Editor
+
+![Blank dialog editor](img/DialogEditor-01.png)
```

Comment:
@cshinn Should "Save" be enabled in this mock?
Should we remove dialog details?  I'm not sure we need it for SUI, but I may be wrong :)